ros2/kachaka_grpc_ros2_bridge/src/converter/ros_header.cpp

```diff
@@ -20,9 +20,10 @@
 namespace kachaka::grpc_ros2_bridge::converter {
 
 void ConvertGrpcHeaderToRos2Header(const kachaka_api::RosHeader& grpc_header,
-                                   std_msgs::msg::Header* ros2_header) {
+                                   std_msgs::msg::Header* ros2_header,
+                                   std::string frame_prefix) {
```

Comment:
ここでも他と同様に const reference にしませんか。
```suggestion
                                   const std::string& frame_prefix) {
```

Reply:
ご指摘の通り，`frame_prefix`は更新されないため`const reference`にした方が良いと考えます．